app.js

```diff
@@ -43,6 +44,7 @@ app.use('/api/v1/token', token)
 app.use('/api/v1/category', category)
 app.use('/api/v1/tag', tag)
 app.use('/api/v1/image', image)
+app.use('/api/v1/sct-treasury', sctTreasury)
```

Comment:
let's not use - or any other symbols in the api endpoints. It could create bad situations for client (bad url encoding).
```suggestion
app.use('/api/v1/scttreasury', sctTreasury)
```